runtime/services/lightd/flora.js

```diff
@@ -48,6 +49,21 @@ Flora.prototype.handlers = {
   }
 }
 
+Flora.prototype.remoteMethods = {
```

Comment:
What's the purpose of this, it seems duplicated with`/lib/component/flora`.

Reply:
It's lightd's flora interface.

Reply:
> It's lightd's flora interface.

What I mean is why don't just call flora's interface?

Reply:
It's flora remote method declaration. It has to be declared on every service like vuid and lightd.